build/azure-pipelines/package-common-create.yml

```diff
@@ -32,6 +32,7 @@ steps:
     BUILD_OUTPUT: $(Pipeline.Workspace)
     BUILD_STAGINGDIRECTORY: $(Build.ArtifactStagingDirectory)
 - task: EsrpCodeSigning@1
```

Comment:
code signing is going to vary widely per package type. Perhaps we could move each signing profile into its own yml and include the appropriate yml based on the package type. For a docker package the included yml could have a no-op step.

Reply:
Agree, I'm about to run into this with the next PR for Windows .msi.